refactored 3 routes change_username, change_password, change_email

mahmoudnasser1561 opened the pull request:
**Description of your changes:**

**moved the following routes change_password, change_email, change_username**

from app.py to their respective files
```change_password.py```  ```change_email.py```  ```change_username.py```

and modified profile template accordingly

mahmoudnasser1561 commented:
please, have a look @pradeeban

pradeeban commented on templates/profile.html:
```diff
@@ -389,19 +389,19 @@ <h3>{{ username }}</h3>
                 </form> -->
             
-                <form action="{{ url_for('change_email') }}" method="POST">
+                <form action="{{ url_for('change_email.change_email') }}" method="POST">
```
A weirdly redundant method. Isn't it?

change_email.change_email, change_password.change_password, and change_username.change_username are all appearing redundant. If you are modularizing, look how these method names should be altered.

perhaps, email.change(), rather than change_email.change_email()?

pradeeban reviewed (changes requested):
Pls check your naming convention to avoid redundancy.

email.py and then change(), rather than change_email.py and change_email()?

mahmoudnasser1561 commented:
@pradeeban 
 I see the redundancy now. I'll update the function names to follow a cleaner structure.
 I will push ASAP.

mahmoudnasser1561 commented:
@pradeeban
please, have a look

I've addressed your feedback regarding the naming conventions. The changes are in these commits:
[1](https://github.com/KathiraveluLab/Beehive/pull/178/commits/26c242676e729ee8e7c414166efe559238fca779)
[2](https://github.com/KathiraveluLab/Beehive/pull/178/commits/e2c3a9df6c0c8996b959969d3bdfbd717bb5101f)
[3](https://github.com/KathiraveluLab/Beehive/pull/178/commits/54b19c6b988d248d73de0808a3745fb23019c1fc)

Please let me know if any further changes are needed.

pradeeban commented:
merged, thanks.
